app/controllers/concerns/potential_spam_user.rb

```diff
@@ -0,0 +1,21 @@
+# This module is the source of truth for what we consider a potential spam user.
+module PotentialSpamUser
+  extend ActiveSupport::Concern
+
+  # I think these should exist in constants even though they are evaluated once
+  # it makes it easier to tweak these variables consistently.
+  # rubocop:disable Rails/RelativeDateConstant
+  AUTH_ACCOUNT_AGE = 50.hours.ago
+  ACCOUNT_NAME_CHARACTER_LENGTH = 30
+  ACCOUNT_AGE = 48.hours.ago
```

Comment:
This will be 48 hours from app start when the constant is assigned, not 48 hours from when the check is performed.  Are we ok with that?

Maybe you could just make them keyword args with defaults on `potential_spam_users`, this way you can even tweak them by playing around in  a Rails console etc.

Reply:
This is true, hadn't really gotten to that yet, as I'm not even sure this is the solution for the issue.

Good idea though, I will probably do that.